src/main/java/edu/cornell/kfs/fp/batch/CreateAccountingDocumentReportItemDetail.java

```diff
@@ -92,10 +93,24 @@ public void setRawDocumentData(String rawDocumentData) {
         this.rawDocumentData = rawDocumentData;
     }
     
+    public String getWarningMessage() {
+        return warningMessage;
+    }
+
+    public void setWarningMessage(String warningMessage) {
+        this.warningMessage = warningMessage;
+    }
+
     public void appendErrorMessageToExistingErrorMessage(String additionalErrorMessageContent) {
         this.setErrorMessage((ObjectUtils.isNotNull(this.getErrorMessage()) ? this.getErrorMessage() : KFSConstants.EMPTY_STRING)
                 + KFSConstants.NEWLINE + KFSConstants.NEWLINE + additionalErrorMessageContent);
          
     }
+    
+    public void appendWarningrMessageToExistingWarningMessage(String additionalWarningMessageContent) {
```

Comment:
Minor typo in method name: "appendWarningrMessage..." --> "appendWarningMessage..."

Reply:
ok